charts/artifact-hub/values.yaml

```diff
@@ -105,6 +105,7 @@ hub:
     enabled: true
     annotations:
       kubernetes.io/ingress.class: nginx
+    defaultBackend: true
```

Comment:
In addition to the default value, we'd also need to:

- [ ] Add and document the new entry in the `values.schema.json` file (please let's mark this new field as required in the corresponding section)
- [ ] Bump the chart version in the `Chart.yaml` file to `1.11.1-2` (otherwise the chart linter will complain)

Reply:
Changes were made.